generate-stackbrew-library.sh

```diff
@@ -35,45 +35,49 @@ outputTag() {
 }
 
 for path in "${paths[@]}"; do
-	echo
+	for variant in alpine ''; do
```

Comment:
Can we switch this to list the non-Alpine images first? This file controls the order they show up in the generated docs on Docker Hub.